src/Microsoft.ML.Mkl.Components/OlsLinearRegression.cs

```diff
@@ -24,16 +24,16 @@
     OlsTrainer.LoadNameValue,
     OlsTrainer.ShortName)]
 
-[assembly: LoadableClass(typeof(OrdinaryLeastSquaresRegressionModelParameters), null, typeof(SignatureLoadModel),
+[assembly: LoadableClass(typeof(OlsModelParameters), null, typeof(SignatureLoadModel),
```

Comment:
>OlsModelParameters [](start = 32, length = 18)

So, general .NET naming guidelines seem to suggest that we avoid acronyms. That is, `OrdinaryLeastSquaresRegressionModelParameters` is right, and `OlsTrainer` is wrong. Similar with PCA. Have we made a conscious decision to discard the .NET guidelines in this case? If we have, could I see where that discussion happened so I can better understand the rationale? It is not in the linked issue, but may have occurred somewhere else.Thanks.

Reply:
I think the discussion is in this thread: #2762

Reply:
The summary is here : https://github.com/dotnet/machinelearning/issues/2762#issuecomment-471845715

Reply:
Ah, all right, thanks for linking the relevant issues.